### ISSUE TITLE
LCD: Make the text left-aligned

### ISSUE THREAD

**Issue by numberZero** [minetest-mods/digilines#41]
Center alignment annoys. It is nice for signs, but the LCD should avoid that.
Also, the LCD dimensions (in letters) should be documented.

**Comment by TangentFoxy** [comment]
Agreed. For now, if you hadn't seen it, the dimensions are 12x5 characters, and there is automatic word-wrapping it seems. (https://github.com/minetest-mods/digilines/blob/master/lcd.lua#L27)

**Comment by DS-Minetest** [comment]
There are other mods which add other lcds, eg.: https://forum.minetest.net/viewtopic.php?t=20794

**Comment by PeterNerlich** [comment]
With #64, you will have the option to use multiple spaces or characters not in the very limited charset (like `´` or `ä`) to control the spacing of characters. For left aligned text, this is still just a workaround, but it at least will be possible then.

**Comment by PeterNerlich** [comment]
#64 is merged, you can now just add spaces to the right of the line until it is full:

    local line_width = 12
    local to_display = "fill up!"
    local pad_right = function(s, times, char)
      char = char or " "
      return tostring(s) .. string.rep(char, times)
    end
    digiline_send('screen', pad_right(to_display, line_width - string.len(to_display)))

This will send `fill up!    ` (with 4 spaces, thank you very much github) to `screen` and any LCD set to this channel will display `fill up!` all the way to the left.
